### DISCUSSION TITLE
Missing Visitors when Filtering for Page (clicking on a page in Top Pages report )?

### DISCUSSION THREAD

**Discussion by Tick20045** [plausible/analytics#1827] [Self-Hosted Support]
Is there any reason why when clicking on a page in the Top Pages report, the visitors from the Sources panel does not add-up to the total visitors to that page?  (I expanded the box by clicking on details and added up the top sources manually)

This only happens for the Page filter. All the other ones are fine.

**Comment by shrinkydinky** [comment]
I've seen this problem too on certain pages. Total visitors for this one over the past week was 54, but source traffic only comes to 23 (direct traffic is missing from the screenshow but was 5).
![Screenshot 2022-07-11 at 11 58 19](https://user-images.githubusercontent.com/4678600/178250001-c4c834da-4035-4440-a413-9f47bd6d16ea.png)

Any ideas?

**Reply by Tick20045** [reply]
Anyone else have any ideas ... this seems pretty big! It doesn't happen on the hosted version

**Reply by Vigasaurus** [reply]
This is working how it's intended indeed, from what I can tell. 

The way it works is that sources shows the number of folks who started their session the page which you have filtered on, as such if you add up the total number of visitors from the sources view, it will line up with the number of visitors who started on that page (from the entry pages report). The number in top pages however, also includes the number of people who may have come to that page as their 2nd+ pageview in the session.
